app/services/bolognese/writers/ris_writer_behavior.rb

```diff
@@ -74,36 +74,38 @@ def ris
           expand_nested_and_prepare(hash)
         end
 
-        # Expand nested arrays, remove any blank entries and expand into a RIS formatted string
-        def expand_nested_and_prepare(hash)
-          hash
-            .compact
-            .map do |k, v|
-              if v.is_a?(Array)
-                # `dup` string for different object ID allowing duplicate keys in compare_by_identity hash
-                v.map { |vi| "#{k.dup}  - #{vi}" if vi.present? }.compact.join(RIS_DELIMITER)
-              else
-                "#{k}  - #{v}"
-              end
-            end.join(RIS_DELIMITER)
-        end
+        protected
 
-        def secondary_titles
-          Array.wrap(parse_attributes(meta["alt_title"])) + Array.wrap(parse_attributes(meta["book_title"]))
-        end
+          # Expand nested arrays, remove any blank entries and expand into a RIS formatted string
+          def expand_nested_and_prepare(hash)
+            hash
+              .compact
+              .map do |k, v|
+                if v.is_a?(Array)
+                  # `dup` string for different object ID allowing duplicate keys in compare_by_identity hash
+                  v.map { |vi| "#{k.dup}  - #{vi}" if vi.present? }.compact.join(RIS_DELIMITER)
```

Comment:
There are two spaces in the string before the "-". The same happens two lines below.

Just a sanity check.

Reply:
Thanks @okeen. Thats actually part of the RIS format specificiation, dont ask me why :)